RNTester/Podfile

```diff
@@ -1,48 +1,28 @@
 platform :ios, '9.0'
 
+require_relative '../scripts/autolink-ios'
```

Comment:
maybe this makes more sense (as a user of react native) as like `require_relative '../node_modules/react-native/scripts/cocoapods'`?